metals/src/main/scala/scala/meta/internal/metals/JavaBinary.scala

```diff
@@ -30,10 +30,10 @@ object JavaBinary {
       .flatMap(home =>
         List(binaryName, binaryName + ".exe").map(home.resolve("bin").resolve)
       )
-      .map { a =>
-        scribe.info(s"java binary path: $a ${a.exists}")
-        a
-
+      .map { path =>
+        scribe.info(s"""java binary path: $path
```

Comment:
Actually, I think I should remove this :/ That was mainly for debugging and it's not really that useful.

Reply:
Sure, I change the `info` to `debug`.

Reply:
Thanks!